src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarPartnerInfo.tsx

```diff
@@ -14,11 +14,16 @@ import { RouterLink } from "v2/System/Router/RouterLink"
 import { useInquiry } from "v2/Components/Inquiry/useInquiry"
 import { useFeatureFlag } from "v2/System/useFeatureFlag"
 import { AnalyticsSchema, useTracking } from "v2/System/Analytics"
+import styled from "styled-components"
 
 export interface ArtworkSidebarPartnerInfoProps {
   artwork: ArtworkSidebarPartnerInfo_artwork
 }
 
+const PartnerContainer = styled(Box)`
+  word-break: break-word;
```

Comment:
This helps with long names on small viewports. For example:

**With word break:**

<img width="319" alt="image" src="https://user-images.githubusercontent.com/265560/165750187-8d3f730b-8601-4059-b2a4-c9beefffbefa.png">


**Without word break:**

<img width="322" alt="image" src="https://user-images.githubusercontent.com/265560/165750075-6d4293d5-19b1-4397-9acd-5f9cb469c789.png">